MarketplacePage/GettingStarted.txt

```diff
@@ -0,0 +1,13 @@
+<p>Use the <code>pitest</code> plugin, such as "solidsoft", in the config file.</p>
```

Comment:
I would specifically mention the `pitest-gradle-plugin` and the  `pl.droidsonroids.pitest` as examples